pkg/app/server/service/webservice/service.proto

```diff
@@ -65,6 +65,9 @@ service WebService {
     rpc UpdatePipedDesiredVersion(UpdatePipedDesiredVersionRequest) returns (UpdatePipedDesiredVersionResponse) {
         option (model.role).project_role = ADMIN;
     }
+    rpc RestartPiped(RestartPipedRequest) returns (RestartPipedResponse) {
+        option (model.role).project_role = ADMIN;
```

Comment:
I guess the EDITOR role is enough to make changes like this

Reply:
I fixed it!

Reply:
@khanhtc1202 @Szarny I prefer `ADMIN` over `EDITOR`.
If only looking from the permission, `EDITOR` is enough to do.
But from the point of responsibility and management, the Admin who created those Piped, who is managing those Piped. And `Restart` is just one of their management operations.

Reply:
I am not familiar with PipeCD Role yet, but I think it would be more natural to revert to ADMIN if the authority for Piped registration and version up is tied to ADMIN role.

Reply:
I don't think I can agree with that 🤔 In my imagination, there will be only one ADMIN for each team/project and several EDITOR (the product devs). Then it would be nice if the EDITOR - who is in charge of deployment can handle restart piped or not by themself. And besides, I think this function does not that powerful or dangerous for the deployment of pipecd application, so EDTIOR is good to me. Happy to hear other thoughts 😄

Reply:
> there will be only one ADMIN for each team/project

Actually, many projects are having multiple admins.
Even in the case of 1 admin, I don't think that the number of admins matters here.

> Then it would be nice if the EDITOR - who is in charge of deployment can handle restart piped or not by themself
>this function does not that powerful or dangerous for the deployment of pipecd application

As our current design, Piped is installed and managed by ADMIN.
EDITOR users only care about the deployment. They don't even care about the Piped status. So why should we give that unneeded ability to them?

That operation is not so dangerous but at least we shouldn't give something to people who don't want it. Besides that, we can limit the miss operation from them.

So if there is no special reason to give that operation to EDITOR, we should keep our consistency as current: all Piped operations belong to ADMIN.
https://github.com/pipe-cd/pipecd/blob/master/pkg/app/server/service/webservice/service.proto#L40-L67


These are my points. But TBH, I don't have a strong opinion. I feel fine for both EDITOR and ADMIN. 👍

Reply:
Thanks for the clarification, after seeing the current piped operations and roles required to do them, I think it would be better to make that ADMIN role required operation (RestartPiped should be as same as UpdatePipedDesiredVersion and Enable/DisablePiped) as you said 👍

Reply:
👍

Reply:
cc @Szarny

Reply:
I updated it to ADMIN :+1: